### PR TITLE
fix the client incorrectly exiting on windows

### PR DESCRIPTION
**Pull request by rogerwim**

### CODE REVIEW

**Review by sverben — APPROVED** [review]

**Comment by rogerwim** [comment]
@sverben could you also merge this?, i cannot

**Comment by sverben** [comment]
> @sverben could you also merge this?, i cannot

I cannot either

File: client.py [hunk 1]
```diff
@@ -22,7 +23,11 @@ def __init__(self, ip):
 
     def on(self, key):
         if hasattr(key, "char"):
-            if key.char in self.keys:
+            if key.char == "k":
+                self.sock.close()
+                _exit(0)
+                exit(0)
```
Comment (Jelmerro) [inline client.py:29]: @rogerwim Do we really need both exit functions?

Comment (rogerwim) [reply client.py]: already fixed

**Review by Jelmerro — APPROVED** [review]

**Pull request merged**